src/main/views/08_vice_analyses.sql

```diff
@@ -0,0 +1,54 @@
+SET search_path = public, pg_catalog;
+
+---
+--- View containing info needed by VICE for its analysis listing.
+---
+CREATE OR REPLACE VIEW vice_analyses AS
+      SELECT j.id,
+             j.job_name,
+             j.app_name,
+             j.subdomain,
+             j.start_date,
+             j.end_date,
+             j.status,
+             u.id AS user_id,
+             u.username,
+             j.job_description,
+             j.app_id,
+             j.result_folder_path,
+             j.planned_end_date,
+             a.description AS app_description,
+             a.edited_date AS app_edited_date,
+             l.id AS tool_id,
+             l.name AS tool_name,
+             l.description AS tool_description,
+             l.version AS tool_version,
+             c.working_directory,
+             c.entrypoint,
+             c.uid,
+             c.min_cpu_cores,
+             c.max_cpu_cores,
+             c.memory_limit,
+             c.pids_limit,
+             c.skip_tmp_mount,
+             p.container_port,
+             i.id AS image_id,
+             i.name AS image_name,
+             i.tag AS image_tag,
+             i.url AS image_url,
+             s.step
+      FROM jobs j
+      JOIN users u ON j.user_id = u.id
+      JOIN job_types t ON j.job_type_id = t.id
```

Comment:
It doesn't look like any info is used from the `job_types` table, so I think this JOIN can be dropped.

Reply:
Good catch